src/classes/modules/java.base/java/util/Calendar.java

```diff
@@ -0,0 +1,3279 @@
+package java.util;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.OptionalDataException;
+import java.io.Serializable;
+import java.security.AccessControlContext;
+import java.security.AccessController;
+import java.security.PermissionCollection;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import java.security.ProtectionDomain;
+import java.text.DateFormat;
+import java.text.DateFormatSymbols;
+import java.time.Instant;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import sun.util.calendar.ZoneInfo;
+import sun.util.locale.provider.CalendarDataUtility;
+import sun.util.locale.provider.TimeZoneNameUtility;
+
+public abstract class Calendar implements Serializable, Cloneable, Comparable<Calendar> {
```

Comment:
Remove unnecessary functionality, as we don't want to support the full class if that is not needed.
While there are applications verified with JPF that use some basic date handling, in-depth date handling code would be an unusual choice for JPF (according to my experience). Typical JPF-verified applications are concurrent or have undetermined inputs, but most business logic is tested using normal unit tests on the regular JVM.

Reply:
Done, only necessary functionality retained now